### DISCUSSION TITLE
Runpod not working with this repo anymore

### DISCUSSION THREAD

**Discussion by leonnn1** [JoePenna/Dreambooth-Stable-Diffusion#89] [Q&A] [answered]
I wonder why, but runpod is not working with this repo unofrtunally, this one gave me the best dreambooth results, does anyone knows a way to make it run?

**Comment by awildcatai** [comment]
I should have noted my errors before closing the pod, but I ran into the same issue. 

RuntimeError: CUDA out of memory was occurring during the training process for me, I triple checked I was definitely connected to a 3090 with 24GB VRAM. 

Also receiving this error throughout the environment creation - WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behavior with the system package manager. It is recommended to use a virtual environment instead.

**Reply by Orlandox683** [reply]
https://www.kris.art/runpoddreambooth

maybe you can fin something here

**Reply by leonnn1** [reply]
I tried to follow your link and it worked! thank you

**Comment by leonnn1** [comment, marked as answer]
Ok the problem is solved, just create a new terminal, then type ps aux and kill the number of the process that has webui.py running, thank you Orlandox683 for the help!
